hack/update-codegen.sh

```diff
@@ -51,7 +51,7 @@ EXTERNAL_INFORMER_PKG="k8s.io/client-go/informers" \
     k8s.io/api \
     "${K8S_TYPES}" \
     --go-header-file ${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt \
-    --force-genreconciler-kinds "Namespace,Deployment,Secret,Pod,CronJob,NetworkPolicy"
+    --force-genreconciler-kinds "Namespace,ConfigMap,Deployment,Secret,Pod,CronJob,NetworkPolicy"
```

Comment:
cc @dprotaso 

We tried to do this one-off downstream first, but there isn't a way to *just* run reconciler generation, so we'd also need to fork the `foo_expansion.go` crap that K8s clients require.

@n3wscott is offline this week, but maybe when he's back we can explore whether some of the generators can be toggled independently 🤔

Reply:
what else does it run/generate?

Reply:
Right now everything we codegen is one giant thing, so all of the injection stuff needs to be regenerated to change any of it.  So all of the injection stuff would need to be regenerated :(